src/Nethermind/Nethermind.Blockchain/BlockTree.cs

```diff
@@ -153,7 +153,7 @@ private void RecalculateTreeLevels()
         private void LoadBestKnown()
         {            
             long headNumber = Head?.Number ?? _syncConfig.PivotNumberParsed;
-            long left = Math.Max(_syncConfig.PivotNumberParsed, headNumber);
+            long left = Math.Max(LowestInsertedHeader?.Number ?? 0, headNumber);
             long right = headNumber + BestKnownSearchLimit;
```

Comment:
if lowest inserted header is non null non zero then pivot should be higher and a better best known on the left. maybe Head?.Number ?? Math.Max(Pivot, LowestInsertedHeader))?